packages/api/src/external/aws/app-config.ts

```diff
@@ -108,7 +108,17 @@ export async function getCxsWithNoWebhookPongFeatureFlagValue(): Promise<string[
 }
 
 export async function getOidsWithIHEGatewayV2Enabled(): Promise<string[]> {
-  return getCxsWithFeatureFlagEnabled("oidsWithIHEGatewayV2Enabled");
+  return Config.isDev()
+    ? Config.getOidsWithIHEGatewayV2Enabled().split(",")
+    : getCxsWithFeatureFlagEnabled("oidsWithIHEGatewayV2Enabled");
+}
+
+export async function getCxsWithIHEGatewayV2Enabled(): Promise<string[]> {
+  if (Config.isDev()) {
```

Comment:
Shouldnt this work like `isEnhancedCoverageEnabledForCx` in the sense that we dont need to check if its dev this will just get configs from staging.

Reply:
I believe Raf had suggested to me to do it this way and model it after getE2eCxIds

Reply:
Yeah, the code I've seen that used a dedicated env var for local had that check after getting the FF, which is not great - it makes the code that uses the FF with more plumbing logic than needed.

There's a benefit to use the `dev` environment on AppConfig so we can test FFs local, but OTOH it makes one's local development dependent on the cloud env - and this is bad because:
- it's shared, two people working on `dev` environment on AppConfig now are racing for the same resource;
- relies on internet to do basic stuff like running the app on one's local computer.

Reply:
But I agree we need a standard here, otherwise its not obvious whether one should set a FF's value on AppConfig or on the local `.env` file.